template-node/exercises/special-relativity-example.yml

```diff
@@ -0,0 +1,13 @@
+---
+uid: special-relativity-example
+title: Special Relativity Example Problem
+type: MultipleChoice
+options:
+  text: Who is known for $E=mc^2$?
+  correctAnswers: 
+    - Einstein
+  incorrectAnswers: 
```

Comment:
I propose that we support the ability to comment on incorrect answers.  If we are using distractor answers in MC questions, notation on why the answers are distractors is potentially critical.